schemas/api/errors.schema.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "$id": "https://fdc3.finos.org/schemas/next/errors.schema.json",
+    "$id": "https://fdc3.finos.org/schemas/next/api/errors.schema.json",
     "definitions": {
         "OpenError": {
             "type": "string",
```

Comment:
missing new error types proposal adds? We'll need to add them into the typescript source for the API too